lineman/app/components/navbar/navbar.coffee

```diff
@@ -16,7 +16,7 @@ angular.module('loomioApp').directive 'navbar', ->
       $scope.selected = component.page
 
     $scope.unreadThreadCount = ->
-      ThreadQueryService.filterQuery('show_unread', queryType: 'inbox').length()
+      ThreadQueryService.filterQuery(['show_unread', 'only_threads_in_my_groups'], queryType: 'inbox').length()
```

Comment:
Just had an idle thought about how should somehow extract this out somehow so that `inbox_page_controller#25` isn't doing the same thing. Not important for this PR.